dotcom-rendering/docs/architecture/030-use-percy-for-e2e-visual-regression-testing.md

```diff
@@ -0,0 +1,62 @@
+# Use Percy/Cypress for integration-style visual regression testing
+
+## Status: Proposed
```

Comment:
Once it’s merged in… it’s accepted, isn’t it? I’m not sure what the purpose of “status” is actually…

```suggestion
## Status: Accepted
```

Reply:
that's a good question! Initially I'd been thinking that it would be merged as a proposal, but given the feedback so far it seems like it makes sense to just add it as an accepted proposal. I'll add a `Decision` section today and then should be able to merge it unless anything else comes up